lib/licenses.nix

```diff
@@ -867,6 +867,22 @@ in mkLicense lset) ({
     fullName = "Lucent Public License v1.02";
   };
 
+  lucidaBitmapFonts = {
+    spxId = "Lucida-Bitmap-Fonts";
+    fullName = "Lucida Bitmap Fonts License";
+    # We consider this as free which is debatable, however
+    # > use […] in individual and commercial software
+    # usage in software implies redistribution
+    # > Users are granted […] to use the trademark […] if the fonts […] are not modified […].
+    # inplies that modification is allowed
+    # Other Opinions:
+    # - SPDX Lists the license (meaning it is free according to their inclusion policy)
+    # - Fedora and other Linux Distributions allow this License:
+    # - https://docs.fedoraproject.org/en-US/legal/allowed-licenses/#_allowed_font_licenses (accessed 2024-06-27)
+    # - Wikipedia says the license is "commercial": https://en.wikipedia.org/w/index.php?title=Lucida&oldid=1182121399
```

Comment:
In the Wikipedia article this is expanded on as follows:

> Lucida is still licensed commercially through the TUG store[5] as well through their own web store.[6] The fonts are occasionally updated.

Indeed, the TUG site [will license you OpenType and Type 1](https://www.tug.org/store/lucida/index.html) Lucida fonts which are presumably not available under a FOSS licence. However, being based on the same typeface design does not mean that the copyright in the fonts is the same; indeed, in the US typeface designs and bitmap fonts are not copyrightable at all, whereas complex vector fonts are treated as software and are eligible for copyright protection.

I don’t think Wikipedia is expressing any opinion on the licence text of this specific distribution of Lucida bitmap fonts; it does not mention the bitmap fonts or their inclusion in X11 at all.